libs/ngworker/lumberjack/http-driver/src/lib/http-driver-root.module.ts

```diff
@@ -37,7 +37,6 @@ export class HttpDriverRootModule {
     @SkipSelf()
     @Inject(HttpDriverRootModule)
     maybeNgModuleFromParentInjector: HttpDriverRootModule = null as any
-    // tslint:enable: no-any no-null-keyword
```

Comment:
Why was this removed?
```suggestion
    // tslint:enable: no-any no-null-keyword
```